server/src/controllers/food.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from "express";
```

Comment:
This PR is for food services, so you didn't need to implement controller files.

We basically won't need any routes other than get (since all database modifications will be done by the database). I should've clarified that so that's my bad.

I'll still add feedback for your work, though. Be sure to reflect on it when working on the food controller ticket.

Reply:
@Remos96